src/main/java/com/minecolonies/colony/buildings/BuildingMiner.java

```diff
@@ -19,24 +19,75 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * The miners building.
+ */
 public class BuildingMiner extends AbstractBuildingWorker
 {
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)